languages/csharp/exercises/concept/numbers-signedness/NumbersUnsignedTest.cs

```diff
@@ -0,0 +1,24 @@
+using Xunit;
+
+public class AssemblyLineTest
```

Comment:
Are we going with test class names being singular or plural (Test or Tests)?
```suggestion
public class LightDistanceTest
```

Reply:
This use the C# track's current test naming convention (singular): https://github.com/exercism/csharp/blob/master/exercises/all-your-base/AllYourBaseTest.cs#L6

I agree that it is a bit odd, but it would require updating all the existing exercises too (which is totally do-able).

Reply:
Let's make them plural :)
https://github.com/exercism/csharp/pull/1364